torntools/scripts/content/stockexchange/ttStockexchange.js

```diff
@@ -504,3 +507,86 @@ function showTotalPortfolioValue() {
 		);
 	}
 }
+
+function hideStockBlocks() {
+	if (hide_stock_blocks.length) {
+		for (let hideStockBlock of hide_stock_blocks) {
+			doc.find(`div.stock-main-wrap li.item.item-wrap[data-stock=${hideStockBlock.toLowerCase()}]`).style.display = "none";
+		}
+		let rawHTML =
+			'<div class="info-msg-cont gray border-round m-top10"><div class="info-msg border-round" style="background-color: #627e0d;"><i class="info-icon"></i><div class="delimiter"><div class="msg right-round" style="background-color: #627e0d;color: #627e0d;">Some stock blocks have been disabled by TornTools. Please re-enable them in Settings.</div></div></div></div>';
```

Comment:
I would also add some bottom margin

Reply:
That is what the class `info-msg msg` does. It already adds some padding so I don't think its necessary.

Reply:
That's not what I noticed, will test again

Reply:
Okay, might have seem strange behavior before